product_available_by_warehouse/models/product.py

```diff
@@ -40,3 +40,11 @@ def get_product_available_by_warehouse(self):
                 'outgoing_qty': outgoing_qty,
             }
         return res
+
+    @api.multi
+    def _get_domain_locations(self):
```

Comment:
In help of `qty_available` field:

> In a context with a single Warehouse, this includes goods stored in the Stock Location of this Warehouse, or any of its children. stored in the Stock Location of the Warehouse of this Shop,  or any of its children Otherwise, this includes goods stored in any Stock Location  with 'internal' type.